Marlin/src/module/tool_change.cpp

```diff
@@ -517,14 +520,14 @@ void fast_line_to_current(const AxisEnum fr_axis) { _line_to_current(fr_axis, 0.
 
     DEBUG_POS("Start ST Tool-Change", current_position);
 
-    current_position.x = placexpos;
+    current_position.x = placexpos + offset_x;
```

Comment:
this line should be changed to something like this:
```
current_position.x = SUM_TERN(HAS_HOTEND_OFFSET, placexpos, offset_x);
```
... Changes below need a similar SUM_TERN treatment.